app/controllers/red_line_boundary_change_requests_controller.rb

```diff
@@ -0,0 +1,44 @@
+class RedLineBoundaryChangeRequestsController < ApplicationController
+  before_action :set_planning_application, only: %i[new create show]
+
+  def new
+    @red_line_boundary_change_request = @planning_application.red_line_boundary_change_requests.new
+  end
+
+  def show
+    @red_line_boundary_change_request = @planning_application.red_line_boundary_change_requests.find(params[:id])
+  end
+
+  def create
+    @red_line_boundary_change_request = RedLineBoundaryChangeRequest.new(new_geojson: red_line_boundary_change_request_params[:new_geojson],
+                                                                         planning_application_id: @planning_application.id,
+                                                                         reason: red_line_boundary_change_request_params[:reason])
```

Comment:
we don't need the `red_line_boundary_change_request_params` method here, it's just calling the params. There's not mass assignment happening.

Reply:
I've changed it to use mass assignment as easier